docs/tutorials/unbound-metrics.md

```diff
@@ -0,0 +1,140 @@
+
+As with all data sources, Netdata can auto-detect Unbound servers if you installed them using the standard installation
+procedure.
+
+Regardless of whether you're connecting to a local or remote Unbound server, you need to be able to access the server's
+`remote-control` interface via an IP address, FQDN, or Unix socket.
+
+To set up the `remote-control` interface, you can use `unbound-control`. First, run `unbound-control-setup` to generate
+the TLS key files that will encrypt connections to the remote interface. Then add the following to the end of your
+`unbound.conf` configuration file. See the [Unbound
+documentation](https://nlnetlabs.nl/documentation/unbound/howto-setup/#setup-remote-control) for more details on using
+`unbound-control`, such as how to handle situations when Unbound is run under a unique user.
+
+```conf
+# enable remote-control
+remote-control:
+    control-enable: yes
+```
+
+Next, make your `unbound.conf`, `unbound_control.key`, and `unbound_control.pem` files readable by Netdata.
+
+```bash
+sudo setfacl -m user:netdata:r unbound.conf
+sudo setfacl -m user:netdata:r unbound_control.key
+sudo setfacl -m user:netdata:r unbound_control.pem
+```
+
+Finally, ensure that your `unbound.conf` file is indented using spaces, not tabs, as our collector can't parse a
+configuration file with tabs. There are many ways to accomplish this, but here is one option:
+
+```bash
+sed -i $'s/\t/    /g' *.conf
+```
+
+Finally, take note whether you're using Unbound in _cumulative_ or _non-cumulative_ mode. This will become relevant when
+configuring the collector.
+
+## Configure the Unbound collector module
+
+To use the Go version of the Unbound collector, you need to explicitly enable it. Open your `go.d.conf` configuration
+file.
+
+```bash
+cd /etc/netdata/ # Replace with your Netdata configuration directory, if not /etc/netdata/
+./edit-config python.d.conf
+```
+
+Find the `unbound` line, uncomment it, and set it to `unbound: yes`.
+
+To configure Netdata's Unbound collector module, navigate to your Netdata configuration directory (typically at
+`/etc/netdata/`) and use `edit-config` to initialize and edit your Unbound configuration file.
+
+```bash
+cd /etc/netdata/ # Replace with your Netdata configuration directory, if not /etc/netdata/
+sudo ./edit-config go.d/unbound.conf
+```
+
+The file contains all the global and job-related parameters. The `name` setting is required, and two Unbound servers
+can't have the same name.
+
+The collector supports both cumulative and non-cumulative modes. Visit Unbound's [statistics configuration
+documentation](https://www.nlnetlabs.nl/documentation/unbound/howto-statistics/) for details on enabling cumulative
+mode, if you're interested.
+
+> It is important you know whether your Unbound server is running in cumulative or non-cumulative mode, as a conflict
+> between modes will create incorrect charts.
+
+At this point, you're ready to edit the `unbound.conf` file according to your needs.
```

Comment:
stock unbound.conf contains 1 predefined job

See
https://github.com/netdata/go.d.plugin/blob/e198258d6d50c46e8299c5e961ac7c0b89f6ad94/config/go.d/unbound.conf#L127-L136

If the user did

```
sudo setfacl -m user:netdata:r unbound.conf
sudo setfacl -m user:netdata:r unbound_control.key
sudo setfacl -m user:netdata:r unbound_control.pem
```

netbound collector will work after restart of netdata service

Reply:
If it is about configuring jobs in general, then nvm 😄